src/deserialize_sequence.cc

```diff
@@ -289,12 +290,14 @@ void load_sequence_parameters(const std::filesystem::path& folder, Sequence& seq
                 sequence.set_timeout(parse_timeout(keyword.substr(11)));
             else if (gul14::starts_with(keyword, "-- tags:"))
                 sequence.set_tags(parse_tags(keyword.substr(8)));
+            else if (gul14::starts_with(keyword, "-- autorun:"))
+                sequence.set_autorun(parse_autorun(keyword.substr(11)));
             else
                 step_setup_script += (line + '\n');
-        }
     }
 
     sequence.set_step_setup_script(step_setup_script);
+    }
```

Comment:
This brace is an artifact from the Gitlab diff view... right?

Reply:
Yes, it is. Stupid web view.

Reply:
Oops, you are right: the additional indentation is missing but you already merged it ...  :-/  For the next PR someone should take care about this (maybe I will do it :-).